.github/pull-request-template.md

```diff
@@ -0,0 +1,18 @@
+## Summary
+
+This PR does...
+
+## Related Issue
+
+<!-- 
+If the Pull request closes/fixes/resolves an issue
+Use Resolves #xx or Closes #xx or Fixes #xx instead
+WITHOUT ANY BRACKETS (It won't work if there are
+brackets)
+-->
+
+*  Issue [#xx]
```

Comment:
I made a small change since Github magic words don't work when there are brackets (from what I've tested so far anyway)

Reply:
_Force pushed to rename commit msg_